src/lib/components/image.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+	import { cn } from "$functions/classnames";
+	import Star from "$icons/shapes/star.svelte";
+	import init, { get_color_thief } from "../../../node_modules/color-thief-wasm-web";
```

Comment:
what do you think bout this import?
reason why I did this:

- showing "unexpected token - export", tried changing vite build config but no hope
- saw many color_thief examples like this

Reply:
.... You sure?

Reply:
lemme see to it